org.ksnip.ksnip.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: org.ksnip.ksnip
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: ksnip
+rename-icon: ksnip
+rename-appdata-file: ksnip.appdata.xml
+rename-desktop-file: ksnip.desktop
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host
+  - --share=network
+  - --own-name=org.kde.*
+  - --talk-name=org.freedesktop.Notifications
+cleanup:
+  - "*.a"
+  - "/include"
+  - "/lib/cmake"
+  - "/lib/mkspecs"
+  - "/mkspecs"
+modules:
+  - name: kColorPicker
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/ksnip/kColorPicker.git
+  - name: kImageAnnotator
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/ksnip/kImageAnnotator.git
+  - name: ksnip
+    buildsystem: cmake-ninja
+    post-install:
+      - install -Dm644 /app/share/pixmaps/ksnip.svg /app/share/icons/hicolor/scalable/apps/ksnip.svg
```

Comment:
This should be fixed upstream as well. `pixmaps` is a deprecated directory and have been the case for a long time.